src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj

```diff
@@ -23,14 +23,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Aquarius.ONE.Models.CSharp" Version="4.7.1" />
+    <PackageReference Include="Aquarius.ONE.Shared.Time" Version="1.1.0" />

```

Comment:
Correct we would need to add the explicit reference again.  I started removing it because I ran into a couple apps that were using an older version and in that case it had to be removed.

Reply:
I agree with Jon, we should remove that dependency of updating time package when there is a new version of protobuf package.